app/main.py

```diff
@@ -2,24 +2,96 @@
 
 
 class IntegerRange:
-    pass
+
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self.private_name = "_" + name
+        print(owner)
+
+    def __get__(self, instance: object, owner: object) -> float:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: object, value: float) -> float | None:
+        if self.min_amount <= value <= self.max_amount:
+            return setattr(instance, self.private_name, value)
+        print(f"Value must be"
+              f"{self.min_amount}...{self.max_amount}")
+        return setattr(instance, self.private_name, None)
 
 
 class Visitor:
-   pass
+
+    def __init__(
+            self,
+            name: str,
+            age: int | None,
+            weight: float | None,
+            height: float | None) -> None:
+
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+
+    def __init__(
+            self,
+            name: str,
+            age: int | None,
+            weight: float | None,
+            height: float | None) -> None:
+
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+
+    age = IntegerRange(min_amount=4, max_amount=14)
+    height = IntegerRange(80, 120)
+    weight = IntegerRange(20, 50)
 
 
 class AdultSlideLimitationValidator(SlideLimitationValidator):
-    pass
+
+    age = IntegerRange(min_amount=14, max_amount=60)
+    height = IntegerRange(120, 220)
+    weight = IntegerRange(50, 120)
 
 
 class Slide:
-    pass
+
+    def __init__(
+            self,
+            name: str,
+            limitation_class: AdultSlideLimitationValidator
+            | ChildrenSlideLimitationValidator
+    ) -> None:
+
+        self.limitation_class = limitation_class
+        self.name = name
+
+    def can_access(self, visitor: Visitor) -> bool:
```

Comment:
after following the above comments you can refactor this method taking into account that a ValueError can be raised. Consider using try/exept blocks